src/Overlay/Modal/Modal.styled.ts

```diff
@@ -43,6 +43,10 @@ export const Dialog = styled.dialog`
     /* Styles for dialogs that carry non-modal behavior */
     &:not(:modal) {
     }
+
+    .alert {
```

Comment:
Is this just a test to see if the classNames were working? The strings provided in classNames most likely won't relate to any CSS here, they will be used for the developers custom CSS.

Reply:
Yes, it was just WIP and to demonstrate that it works. I removed it.